### DISCUSSION TITLE
Template Addition Request: shlink.io

### DISCUSSION THREAD

**Discussion by jijojosein** [Dokploy/dokploy#221] [Ideas]
Would be great to have a link shortener as a template. 

https://shlink.io/ looks great, has a robust API, supports multiple domains, docker image is available. 

https://shlink.io/documentation/
https://shlink.io/documentation/install-docker-image/

Thank you 🙏

**Comment by nktnet1** [comment]
PR here:
- https://github.com/Dokploy/dokploy/pull/1226
